flycheck-elsa.el

```diff
@@ -46,23 +45,65 @@
   :group 'flycheck-elsa
   :type '(repeat regexp))
 
+(defun flycheck-elsa--locate-cask-dir ()
+  "Return dir located Cask file.  If missing, return nil."
+  (when-let (file (locate-dominating-file (buffer-file-name) "Cask"))
+    (file-name-directory file)))
+
+(defun flycheck-elsa--elsa-dependency-p ()
+  "Return non-nil if elsa listed in Cask file dependency."
+  (let ((cask-dir (flycheck-elsa--locate-cask-dir)))
+    (with-temp-buffer
+      (insert-file-contents (expand-file-name "Cask" cask-dir))
+      (let* ((contents (read (format "(%s)" (buffer-string))))
+             (deps (append
+                    (mapcan
+                     (lambda (elm)
+                       (and (eq 'depends-on (car elm)) (list elm))) contents)
+                    (mapcan
+                     (lambda (elm)
+                       (and (eq 'development (car elm))
+                            (mapcan
+                             (lambda (elm)
+                               (and (eq 'depends-on (car elm)) (list elm)))
+                             (cdr elm)))) contents))))
+        (and (delq 'nil (mapcar
+                         (lambda (elm) (string= "elsa" (nth 1 elm))) deps))
+             t)))))                     ; normarize return value
+
+(defun flycheck-elsa--install-elsa ()
```

Comment:
We should not do this.  Users should run `cask install` themselves.

Reply:
OK. I'll revert adding it commit.